lua/llimits.h

```diff
@@ -128,7 +128,7 @@ typedef LUA_UACNUMBER l_uacNumber;
 ** type for virtual-machine instructions
 ** must be an unsigned with (at least) 4 bytes (see details in lopcodes.h)
 */
-typedef unsigned long Instruction;
+typedef unsigned int Instruction;
```

Comment:
If this is a change to make it a fixed-width integer, it should just be `uint32_t`.